bazel/repository_locations.bzl

```diff
@@ -16,6 +16,11 @@ REPOSITORY_LOCATIONS = dict(
             "https://mirror.bazel.build/github.com/bazelbuild/bazel-toolchains/archive/1.1.0.tar.gz",
         ],
     ),
+    build_bazel_rules_apple = dict(
+        urls = ["https://github.com/bazelbuild/rules_apple/archive/b869b0d3868d78a1d4ffd866ccb304fb68aa12c3.tar.gz"],
```

Comment:
@lizan the commit you used here is in the https://github.com/bazelbuild/rules_apple/releases/tag/0.19.0 release of `rules_apple`. Any reason to use the commit vs. the release?

Reply:
Probably this error...
```
ERROR: /home/moderation/.cache/bazel/_bazel_moderation/3002a12f12a06255067bdca8d32b0440/external/envoy/bazel/BUILD:194:1: error loading package '@com_github_grpc_grpc//src/proto/grpc/health/v1': in /home/moderation/.cache/bazel/_bazel_moderation/3002a12f12a06255067bdca8d32b0440/external/com_github_grpc_grpc/bazel/grpc_build_system.bzl: in /home/moderation/.cache/bazel/_bazel_moderation/3002a12f12a06255067bdca8d32b0440/external/build_bazel_rules_apple/apple/ios.bzl: in /home/moderation/.cache/bazel/_bazel_moderation/3002a12f12a06255067bdca8d32b0440/external/build_bazel_rules_apple/apple/internal/testing/ios_rules.bzl: in /home/moderation/.cache/bazel/_bazel_moderation/3002a12f12a06255067bdca8d32b0440/external/build_bazel_rules_apple/apple/internal/testing/apple_test_bundle_support.bzl: in /home/moderation/.cache/bazel/_bazel_moderation/3002a12f12a06255067bdca8d32b0440/external/build_bazel_rules_swift/swift/swift.bzl: in /home/moderation/.cache/bazel/_bazel_moderation/3002a12f12a06255067bdca8d32b0440/external/build_bazel_rules_swift/swift/internal/swift_library.bzl: Unable to load file '@bazel_skylib//rules:common_settings.bzl': file doesn't exist and referenced by '@envoy//bazel:grpc_health_proto'
ERROR: Analysis of target '//source/exe:envoy-static.stripped' failed; build aborted: Analysis failed
```

Reply:
Got to the bottom of this and will follow up with a new PR.

```diff
diff --git a/api/bazel/repository_locations.bzl b/api/bazel/repository_locations.bzl
index 2f29473d1..941b4096e 100644
--- a/api/bazel/repository_locations.bzl
+++ b/api/bazel/repository_locations.bzl
@@ -1,11 +1,11 @@
-BAZEL_SKYLIB_RELEASE = "0.8.0"
-BAZEL_SKYLIB_SHA256 = "2ef429f5d7ce7111263289644d233707dba35e39696377ebab8b0bc701f7818e"
+BAZEL_SKYLIB_RELEASE = "0.9.0"
+BAZEL_SKYLIB_SHA256 = "1dde365491125a3db70731e25658dfdd3bc5dbdfd11b840b3e987ecf043c7ca0"
```
```diff
diff --git a/bazel/repository_locations.bzl b/bazel/repository_locations.bzl
index 85d892335..00b7e7f76 100644
--- a/bazel/repository_locations.bzl
+++ b/bazel/repository_locations.bzl
     ),
     build_bazel_rules_apple = dict(
-        urls = ["https://github.com/bazelbuild/rules_apple/archive/b869b0d3868d78a1d4ffd866ccb304fb68aa12c3.tar.gz"],
-        strip_prefix = "rules_apple-b869b0d3868d78a1d4ffd866ccb304fb68aa12c3",
-        sha256 = "bdc8e66e70b8a75da23b79f1f8c6207356df07d041d96d2189add7ee0780cf4e",
+        sha256 = "7a7afdd4869bb201c9352eed2daf37294d42b093579b70423490c1b4d4f6ce42",
+        urls = ["https://github.com/bazelbuild/rules_apple/releases/download/0.19.0/rules_apple.0.19.0.tar.gz"],
     ),
```

Reply:
I picked this sha/commit from grpc directly, that PR sgtm though.